templates/views/blog/_pagination.html.twig

```diff
@@ -3,19 +3,19 @@
 		<div class="navigation">
 			<ul class="pagination">
 				{% if paginator.hasPreviousPage %}
-					<li class="prev arrow"><a href="{{ path('blog_paginated', {page: paginator.previousPage, author: author, tag: app.request.get('tag')}) }}" rel="previous"><i class="fa fw fa-chevron-left"></i></a></li>
+    				<li class="prev arrow"><a href="{{ path('blog_paginated', {page: paginator.previousPage, author: author, tag: app.request.get('tag'), query: app.request.get('query')}) }}" rel="previous"><i class="fa fw fa-chevron-left"></i></a></li>
 				{% endif %}
 
 				{% for i in 1..paginator.lastPage %}
 					{% if i == paginator.currentPage %}
 						<li class="active"><a>{{ i }}</a></li>
 					{% else %}
-						<li><a href="{{ path('blog_paginated', {page: i, author: author, tag: app.request.get('tag')}) }}">{{ i }}</a></li>
+						<li><a href="{{ path('blog_paginated', {page: i, author: author, tag: app.request.get('tag'), query: app.request.get('query') }) }}">{{ i }}</a></li>
 					{% endif %}
 				{% endfor %}
 
 				{% if paginator.hasNextPage %}
-					<li class="next arrow"><a href="{{ path('blog_paginated', {page: paginator.nextPage, author: author, tag: app.request.get('tag')}) }}" rel="next"><i class="fa fw fa-chevron-right"></i></a></li>
+					<li class="next arrow"><a href="{{ path('blog_paginated', {page: paginator.nextPage, author: author, tag: app.request.get('tag')}), query: app.request.get('query') }}" rel="next"><i class="fa fw fa-chevron-right"></i></a></li>
```

Comment:
```suggestion
					<li class="next arrow"><a href="{{ path('blog_paginated', {page: paginator.nextPage, author: author, tag: app.request.get('tag'), query: app.request.get('query') }) }}" rel="next"><i class="fa fw fa-chevron-right"></i></a></li>
```

Reply:
@MightyM17 Last one, but don't worry I will commit it myself if you don't have time to do it
Also, I put the label Hacktoberfest accepted!

Reply:
Fixed it, my bad for the brackets 😓